apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -338,6 +335,10 @@ const aichatSlice = createSlice({
         }
       }
 
+      if (studentAiCustomizations.modelCardInfo.isPublished) {
```

Comment:
Feels a bit hacky :) LMK if there are better ideas here.

Reply:
Is this needed? I thought since `isPublished` is a property of `modelCardInfo`, it would already be 'reconciled' above.

Reply:
I thought I needed it, but I totally cannot remember why and I think you're right from what I can see...removing!